plugins/wazo-snom/common/common.py

```diff
@@ -156,6 +157,14 @@ def __init__(self, app, plugin_dir, gen_cfg, spec_cfg):
 
     http_dev_info_extractor = BaseSnomHTTPDeviceInfoExtractor()
 
+    def _add_uxm_firmware(self, raw_config):
+        f = glob.glob(self._tftpboot_dir + u'/firmware/snomD7C-*.bin') + glob.glob(self._tftpboot_dir + u'/firmware/snomUXM-*.bin')
```

Comment:
It is unsafe to use `+` to join a path: use `os.path.join` (as on line 171) to concatenate a filepath :smile:

Reply:
done !